cfme/infrastructure/virtual_machines.py

```diff
@@ -261,6 +261,7 @@ def revert_to(self, cancel=False):
     ALL_LIST_LOCATION = "infra_vms"
     TO_OPEN_EDIT = "Edit this VM"
     TO_RETIRE = "Retire this VM"
+    BaseVM.VM_TYPE = "Virtual Machine"
```

Comment:
This is not good. Just VM_TYPE. Otherwise you override it in the base class making all of the subclasses be the same type.